src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -7830,10 +7831,11 @@ public void M(string s)
 @"
 public class P
 {
+    private const string {|Rename:V|} = $"""";
```

Comment:
>private const string {|Rename:V|} = $""""; [](start = 4, length = 42)

It is not clear why this became a field.

Reply:
This one is correct. The string has a constant value, and the IDE prefers to extract a constant if it can do so.

Reply:
> This one is correct. The string has a constant value, and the IDE prefers to extract a constant if it can do so.

A constant doesn't have to be a field.

---
In reply to: [533865977](https://github.com/dotnet/roslyn/pull/49707#discussion_r533865977) [](ancestors = 533865977)

Reply:
I would expect it to remain a local.

---
In reply to: [533866290](https://github.com/dotnet/roslyn/pull/49707#discussion_r533866290) [](ancestors = 533866290,533865977)

Reply:
> I would expect it to remain a local.

That's a question for the IDE team. They already approved this behavior when other tests were updated with non-empty interpolated strings: [`43b2c9c` (#46828)](https://github.com/dotnet/roslyn/pull/46828/commits/43b2c9c942395b539c02edd72be9aeea154f9d02)